applications/nrf_desktop/doc/hids.rst

```diff
@@ -104,7 +104,3 @@ Transport for configuration channel
 
 The HID Service application module works as a transport for the :ref:`nrf_desktop_config_channel` and exchanges the |conf_channel| HID reports over Bluetooth LE.
 The module communicates with the |conf_channel| listeners using ``config_event``.
-
-.. |GATT_HID| replace:: GATT HID Service
-.. |HID_state| replace:: HID state module
```

Comment:
Consider changing references in this file to `|hid_state|` (to improve letter case consistency with tag defined in `shortcuts.txt`)

Reply:
updated :)